extensions/webservices/joomla/site.users.1.1.0.php

```diff
@@ -135,6 +135,16 @@ public function authenticateUser($email)
 
 		$user->user_groups = $userGroups;
 
+		// Set access token to expire for security reasons
+		$accessTokenKey = $app->input->get('access_token');
```

Comment:
access_token can be something different... Can you please get the name from the redCORE config like so... RBootstrap::getConfig('oauth2_token_param_name', 'access_token')